crypto/http/http_client.c

```diff
@@ -355,6 +357,16 @@ int OSSL_HTTP_REQ_CTX_set1_req(OSSL_HTTP_REQ_CTX *rctx, const char *content_type
     return res;
 }
 
+void OSSL_HTTP_REQ_CTX_set_max_response_hdr_lines(OSSL_HTTP_REQ_CTX *rctx,
+                                                  size_t count)
```

Comment:
I wonder if adding this call isn't an overkill. If the limit was set to 1024 for example, would any application really need to override it?